src/main/java/uk/ac/ebi/ena/webin/cli/service/LoginService.java

```diff
@@ -62,30 +71,49 @@ public LoginService(String username, String password, boolean test) {
     }
 
     public String login() {
-        RestTemplate restTemplate = new RestTemplate();
-        restTemplate.setErrorHandler(new DefaultErrorHander(WebinCliMessage.CLI_AUTHENTICATION_ERROR.text()));
         RequestEntity< LoginRequestBody > request =  getAuthRequest("/login");
```

Comment:
I think we should continue to use setErrorHandler and improve the NotFoundErrorHandler. See previous comment.